src/nls/messages_en.json

```
@@ -133,7 +135,7 @@
     "dashboard.importResources.repo.helperText": "The location of the YAML definitions to be applied (Git URLs supported)",
     "dashboard.importResources.repo.invalidText": "Please submit a valid URL",
     "dashboard.importResources.repo.labelText": "Repository URL",
-    "dashboard.importResources.serviceAccount.helperText": "The ServiceAccount that the PipelineRun will run under (from the namespace above)",
+    "dashboard.importResources.serviceAccount.helperText": "The ServiceAccount that the PipelineRun applying resources will run under (from the namespace above). Ensure the selected ServiceAccount (or the default if none selected) has permissions for creating PipelineRuns and for anything else your PipelineRun interacts with, including and any Tekton resources in the Git repository.",
```

```suggestion
    "dashboard.importResources.serviceAccount.helperText": "The ServiceAccount that the PipelineRun applying resources will run under (from the namespace above). Ensure the selected ServiceAccount (or the default if none selected) has permissions for creating PipelineRuns and for anything else your PipelineRun interacts with, including any Tekton resources in the Git repository.",
```

unnecessary 'and'... 'including and any Tekton resources'